src/screens/SearchScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+
+import EditScreenInfo from 'components/EditScreenInfo';
+import { useEffect, useState } from 'react';
+import { getAllTrees } from 'src/database/firebase';
+import { Tree } from '@types';
+import SearchCard from 'src/components/SearchCard';
+import { Searchbar, Title } from 'react-native-paper';
+import ViewContainer from 'src/components/ViewContainer';
+
+export default function SearchScreen() {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [filteredTrees, setFilteredTrees] = useState<Tree[]>([]);
+  const [trees, setTrees] = useState<Tree[]>([]);
+  useEffect(() => {
+    async function getTrees() {
+      const data = await getAllTrees();
+      setTrees(data);
+      setFilteredTrees(data);
+    }
+    getTrees();
+  }, []);
+
+  const onSearchChange = (searchValue: string) => {
+    setSearchQuery(searchValue);
+    // Checking searchValue because searchQuery isn't set instantaneously
+    if (searchValue !== '') {
+      const newFilteredTrees = trees.filter(
+        (tree: Tree) => (tree.name || '').toLowerCase().includes(searchQuery) ||
```

Comment:
nit: @kennethlien I feel like something we're going to encounter is that a lot of our variables will be named `<something>Trees`. This seems redundant and can be reasonably inferred that when we talk about data we mean trees unless otherwise specified, thoughts?

Reply:
We should pre-filter out any null or empty-stringed names from our search results, seems rendering them as cards doesn't make sense.